### PR TITLE
Added supporting SSO logon for consoles

### PR DESCRIPTION
**Pull request by bond95**
If VM has SSO method 'guest_agent', then it will try to logon, if there will be an error, then user get modal window:
image

Fixes: https://github.com/oVirt/ovirt-web-ui/issues/65

### CODE REVIEW

File: src/intl/messages.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ export const messages: { [messageId: string]: MessageType } = {
   bootSequence: 'Boot Sequence',
   bootSequenceTooltip: 'Device order in which VM will search OS for running.',
   cancel: 'Cancel',
+  cantLogonToConsole: 'Can\'t logon to console. Do you want to continue?',
```
Comment (sjd78) [inline src/intl/messages.js]: Perhaps a bit more descriptive:

"SSO logon to the VM's console failed.  Do you want to continue without SSO?"

**Comment by bond95** [comment]
@sjd78 Yes, logon request work for users (tested on brq-dev).

**Comment by bond95** [comment]
ci build please

**Comment by gregsheremeta** [comment]
ci build please

**Comment by bond95** [comment]
- https://jenkins.ovirt.org/job/oVirt_ovirt-web-ui_standard-check-pr/188/artifact/build-artifacts.fc28.x86_64/ovirt-web-ui-1.5.0-0.20181109git19cd4f2.fc28.noarch.rpm

- https://jenkins.ovirt.org/job/oVirt_ovirt-web-ui_standard-check-pr/188/artifact/build-artifacts.el7.x86_64/ovirt-web-ui-1.5.0-0.20181109git19cd4f2.el7.noarch.rpm

**Comment by gregsheremeta** [comment]
ci build please

**Comment by gregsheremeta** [comment]
ci build please

**Comment by gregsheremeta** [comment]
[need to fix the build]

**Comment by sjd78** [comment]
ci build please

**Comment by sjd78** [comment]
> [need to fix the build]

fixed!

**Comment by pnovotny** [comment]
LGTM

**Pull request merged**